components/resolved_ts/src/resolver.rs

```diff
@@ -125,16 +142,23 @@ impl Resolver {
             start_ts,
             self.region_id
         );
+        let bytes = key.as_slice().heap_size();
```

Comment:
why only count the key's heap size

Reply:
Do you mean we should also count timestamp? If so, it's addressed in https://github.com/tikv/tikv/pull/15411/files#diff-d486ab2e0d045721c6940de0097bef32d5c22ea656e1605ef475a731a971cfa4R169


```diff
-        let bytes = key.as_slice().heap_size();
+        let bytes = key.as_slice().heap_size() + std::mem::size_of::<TimeStamp>();